<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>confirmNew</class>
 <widget class="QDialog" name="confirmNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>299</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>confirmNew</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(0, 150, 225);</string>
     </property>
     <property name="text">
      <string>游戏正在进行，您希望做什么？</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="btnNew">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="text">
      <string>退出并重新开始游戏(&amp;N)
本局将作为失败记入统计信息。</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="btnRestart">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="text">
      <string>重新开始这个游戏(&amp;R)
本局将作为失败记入统计信息。</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="btnCancel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="text">
      <string>继续游戏(&amp;K)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
